src/mds_array_manipulation/mds_array_manipulation.py

```diff
@@ -1,3 +1,74 @@
+def argmax(arr, axis=None):
+    """
+    Returns the indices of the maximum values along an axis from given array.
+
+    Parameters
+    ----------
+    arr : numpy.array
+        Input array.
+    axis : int, optional
+        Axis along which to operate. By default, flattened input is used.
+
+    Returns
+    -------
+    indices : int or tuple of ints
+        Indices of the maximum values along the specified axis.
+
+    Notes
+    -----
+    If there are multiple occurrences of the maximum values, the indices
+    corresponding to the first occurrence are returned.
+
+    Examples
+    --------
+    >>> a = np.arange(6).reshape(2,3)
+    >>> a
+    array([[0, 1, 2],
+           [3, 4, 5]])
+    >>> argmax(a)
+    5
+    >>> argmax(a, axis=0)
+    array([1, 1, 1])
+    >>> argmax(a, axis=1)
+    array([2, 2])
+
+    >>> b = np.arange(6)
+    >>> b[1] = 5
+    >>> b
+    array([0, 5, 2, 3, 4, 5])
+    >>> argmax(b)  # Only the first occurrence is returned.
+    1
+    """
+    array = np.array(arr)  # Convert input to a NumPy array, in case of input array is not numpy array 
```

Comment:
We just need the skeleton for now - no actual code: Maybe can move this code to a separate branch to be used for the next milestone?